mix.exs

```diff
@@ -48,7 +53,7 @@ defmodule LiveDebugger.MixProject do
       {:phoenix_live_view, "~> 0.20 or ~> 1.0"},
       {:bandit, "~> 1.6", only: :dev},
       {:phoenix_live_reload, "~> 1.5", only: :dev},
-      {:esbuild, "~> 0.7", only: :dev},
+      {:esbuild, "~> 0.7", only: [:dev, :prod]},
```

Comment:
If we have to depend on `esbuild` we'll have to move it just below LiveView, and probably adjust accepted `esbuild` versions. This should work on as many `esbuild` as possible